docusaurus/docs/Android/03-ui/02-general-customization/02-chatui.mdx

```diff
@@ -289,3 +469,33 @@ ChatUI.dateFormatter = object: DateFormatter {
     }
 }
 ```
+</TabItem>
+
+<TabItem value="java" label="Java">
+
+```java
+ChatUI.setDateFormatter(new DateFormatter() {
+    private DateTimeFormatter dateFormatter = DateTimeFormatter.ofPattern("yy MM dd");
+    private DateTimeFormatter dateTimeFormatter = DateTimeFormatter.ofPattern("HH:mm");
+
+    @NonNull
+    @Override
+    public String formatDate(@Nullable LocalDateTime localDateTime) {
+        if (localDateTime == null) {
+            return "";
+        }
+        return dateFormatter.format(localDateTime);
```

Comment:
<img width="374" alt="Screenshot 2022-10-26 at 08 24 20" src="https://user-images.githubusercontent.com/1286516/198068304-6577b3a6-6598-4d6b-ba1f-1381439505cf.png">

Probably another IDE issue, which is not seeing type correctly in Java =[

Reply:
🤔 works fine on my end